backend/event/models.py

```diff
@@ -160,3 +166,53 @@ class Role(Field, models.Model):
 
     def __str__(self):
         return f"role of {self.person} in {self.letter_action}"
+
+
+class WorldEvent(LettercraftDate, models.Model):
+    """
+    World events are events that are not directly related to a specific letter
+    or letter action, but are relevant to the context of the letters.
+    """
+
+    name = models.CharField(
+        max_length=256,
+        null=False,
+        blank=False,
+        help_text="The name of the event, e.g. 'The Great Fire of London' or 'The Battle of Hastings'.",
+    )
+
+    note = models.TextField(
+        null=False,
+        blank=True,
+        help_text="Additional notes that describe the event and its relevance to the letters.",
+    )
+
+    epistolary_events = models.ManyToManyField(
+        to=EpistolaryEvent,
+        through="EpistolaryEventTrigger",
+        related_name="triggers",
+    )
```

Comment:
Logically, epistolary events can also trigger world events, or epistolary events can trigger other epistolary events. Perhaps this requires a more complex relationship?

Reply:
Ah, good point. I'll make the relationship bidirectional.